apps/web/components/Menu/Menu.tsx

```diff
@@ -94,7 +94,10 @@ export const Menu = ({
       }}
       renderMyPagesButton={(button) => {
         return (
-          <Link href="//minarsidur.island.is/" skipTab>
+          <Link
+            href={activeLocale === 'en' ? '/en/login' : '/innskraning'}
```

Comment:
Consider adding this url to the link resolver to keep internal link resolution consistent

Reply:
This has been changed ✅